StreamChatClient_v3/Controllers/ChangeAggregator_Tests.swift

```diff
@@ -0,0 +1,206 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+@testable import StreamChatClient_v3
+import XCTest
+
+class ChangeAggregator_Tests: XCTestCase {
+    var fakeController: NSFetchedResultsController<NSFetchRequestResult>!
+    var aggregator: ChangeAggregator<TestManagedObject, String>!
+    
+    override func setUp() {
+        super.setUp()
+        // This is just for the delegate calls, we don't use it anywhere
+        fakeController = .init()
+        
+        // We don't have to provide real creator. Let's just simply use the value that gets in
+        aggregator = ChangeAggregator(itemCreator: { $0.testId })
+    }
+    
+    func test_addingItems() {
+        // Set up aggregator callback
+        var callback: [Change<String>]?
```

Comment:
This is `result`, not `callback`

Reply:
fixed in 558be31403cba6f55b8dbd58636527ad181fd39b